components/renku_data_services/namespace/db.py

```diff
@@ -575,6 +575,34 @@ async def _check_dc_slug_not_taken(
                     message=f"The owner already has a data connector with slug {new_slug}, please try a different one"
                 )
 
+        async def _upsert_old_dc_slug(session: AsyncSession, old_dc_slug: schemas.EntitySlugORM) -> None:
```

Comment:
issue: a docstring here why this is needed and what it does so posterity remembers would be nice

Reply:
Done in https://github.com/SwissDataScienceCenter/renku-data-services/pull/724/commits/29f2d9308aa5e32537a88ee05df84db048426f22